02_Development/Display/features/gets_sensor_data.feature

```diff
@@ -7,11 +7,12 @@ Feature: gets data from sensors
         Then the software will determine which tubes have both pressure and airflow sensing
         Then the software will return the state of the sensor to the user.
 
```

Comment:
we may consider adding the feature of indicating when the unit is unplugged. Their is an input on one of the channels of the ADC that measures input voltage. If the input voltage  measured by the ADC indicates 12.7V, we're plugged into the wall. If the input voltage measured by the ADC indicates 12V, we're on battery power.